### PR TITLE
Pull request from Anh Ho, updated parameters.

### PR DESCRIPTION
**Pull request by tuananh1508**

### CODE REVIEW

File: aws-env.dev [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-export MYSQL_DATABASE=bugzilla
-export MYSQL_HOST=auroradb.dev.unee-t.com
-export MYSQL_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MYSQL_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export MYSQL_PORT=3306
-export MYSQL_USER=bugzilla
-export PARAMS_URL=https://raw.githubusercontent.com/unee-t/bugzilla-customisation/master/params/dev-params.json
-export SES_SMTP_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export SES_SMTP_USERNAME=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_USERNAME --query Parameters[0].Value --output text)
-export SES_VERIFIED_SENDER=dev.case@unee-t.com
+export BUGZILLA_ADMIN_KEY_INS=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY --with-decryption --query Parameters[0].Value --output text)
```
Comment (franck-boullier) [inline aws-env.dev]: This should read 
`export BUGZILLA_ADMIN_KEY=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY_INS ...`

File: aws-env.dev [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+export BUGZILLA_URL_INS=https://dashboard-ins.dev.unee-t.com
```
Comment (franck-boullier) [inline aws-env.dev]: This should read 
`export BUGZILLA_URL=https://dashboard-ins.dev.unee-t.com`

**Review by franck-boullier — CHANGES_REQUESTED** [review]
you should NOT change the name of the variables
everything before the `=` sign should stay the same or the script will likely fail.

My understanding of the syntax in this file is:
`export [variable_used_in_the_deployment_script] = [value_for_this_variable]`

you should only change the `[value_for_this_variable]` where needed.
you should not touch the `[variable_used_in_the_deployment_script]` portion.

File: aws-env.dev [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+export MONGO_PASSWORD_INS=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD --with-decryption --query Parameters[0].Value --output text)
+export MONGO_CONNECT_INS="dev-shard-00-00-qxxao.mongodb.net:27017,dev-shard-00-01-qxxao.mongodb.net:27017,dev-shard-00-02-qxxao.mongodb.net:27017/test?ssl=true&replicaSet=Dev-shard-0&authSource=admin"
+export MYSQL_DATABASE_INS=bugzilla
+export MYSQL_HOST_INS=auroradb-ins.dev.unee-t.com
+export MYSQL_PASSWORD_INS=$(aws --profile uneet-dev ssm get-parameters --names MYSQL_PASSWORD --with-decryption --query Parameters[0].Value --output text)
+export MYSQL_PORT_INS=3306
+export MYSQL_USER_INS=bugzilla
+export PARAMS_URL_INS=https://github.com/tuananh1508/bugzilla-customisation/blob/master/params/dev-params.json
```
Comment (franck-boullier) [inline aws-env.dev]: assume that we will run the code from the master <-- use the url for the version of the file in the master repo, not the version of the URL from your fork.

**Comment by franck-boullier** [comment]
# Minor things:

Even though I don't think that this is a major issue at this point, these are notes for future reference:

The following parameters also need to be updated:
- [mailfrom](https://github.com/Unee-T-INS/bugzilla-customisation/blob/d03813902c92ef31e60699ec2046135582cf086d/params/dev-params.json#L52)
- [maintainer](https://github.com/Unee-T-INS/bugzilla-customisation/blob/d03813902c92ef31e60699ec2046135582cf086d/params/dev-params.json#L53)

File: aws-env.dev [hunk 4]
```diff
@@ -1,13 +1,13 @@
-export BUGZILLA_ADMIN_KEY=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY --with-decryption --query Parameters[0].Value --output text)
-export BUGZILLA_URL=https://dashboard.dev.unee-t.com
-export MONGO_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD --with-decryption --query Parameters[0].Value --output text)
+export BUGZILLA_ADMIN_KEY=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY_INS --with-decryption --query Parameters[0].Value --output text)
+export BUGZILLA_URL=https://dashboard-ins.dev.unee-t.com
+export MONGO_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD_INS --with-decryption --query Parameters[0].Value --output text)
 export MONGO_CONNECT="dev-shard-00-00-qxxao.mongodb.net:27017,dev-shard-00-01-qxxao.mongodb.net:27017,dev-shard-00-02-qxxao.mongodb.net:27017/test?ssl=true&replicaSet=Dev-shard-0&authSource=admin"
```
Comment (franck-boullier) [inline aws-env.dev]: This should probably point to the Mongo database that we have created for the Unee-T INS installation

File: aws-env.dev [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+export MYSQL_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MYSQL_PASSWORD_INS --with-decryption --query Parameters[0].Value --output text)
 export MYSQL_PORT=3306
 export MYSQL_USER=bugzilla
-export PARAMS_URL=https://raw.githubusercontent.com/unee-t/bugzilla-customisation/master/params/dev-params.json
-export SES_SMTP_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export SES_SMTP_USERNAME=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_USERNAME --query Parameters[0].Value --output text)
+export PARAMS_URL=https://raw.githubusercontent.com/Unee-T-INS/bugzilla-customisation/master/params/dev-params.json
+export SES_SMTP_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_PASSWORD_INS --with-decryption --query Parameters[0].Value --output text)
+export SES_SMTP_USERNAME=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_USERNAME_INS --query Parameters[0].Value --output text)
 export SES_VERIFIED_SENDER=dev.case@unee-t.com
```
Comment (franck-boullier) [inline aws-env.dev]: if you've created a dedicated SES service for Unee-T INS (which is good) I think you also have to create a new SES_VERIFIED_SENDED too.
maybe you can configure `dev.case.ins@unee-t.com` as an authorized sender in SES?

**Review by franck-boullier — CHANGES_REQUESTED** [review]
almost there!

File: aws-env.dev (hunk 5 above)
Comment (tuananh1508) [reply aws-env.dev]: I can config dev.case.ins@unee-t.com. But we have to create that email and verify it first.

Comment (franck-boullier) [reply aws-env.dev]: I have created the following email addresses as forwarders to the email address osam@unee-t.com:
- dev.ins@unee-t.com: generic email address for the DEV environment of the Unee-T INS installation
- dev.case.ins@unee-t.com: email address for the MEFE side of DEV environment of the Unee-T INS installation
- dev.dashboard.ins@unee-t.com: email address for the BZFE side of DEV environment of the Unee-T INS installation

**Pull request merged**